src/main/java/com/ibm/as400/access/Permission.java

```diff
@@ -885,6 +901,18 @@ public Enumeration<UserPermission> getUserPermissions()
       }
     }
 
+    /**
+     * Returns a list of UserPermission objects.
+     * @return A list of UserPermission objects.
+     *
+     **/
+    public List<UserPermission> getUserPermissionsList() {
+        synchronized (userPermissionsLock_)
+        {
+            return Collections.list(userPermissions_.elements());
```

Comment:
`userPermissions_` is already of type `Vector` which is already a `List`. 
So this implementation could simply be
```java
return userPermissions_;
```

Reply:
thanks, updated the code